views/006_config_views.sql

```diff
@@ -3,6 +3,66 @@ DROP VIEW IF EXISTS configs CASCADE;
 
 DROP FUNCTION IF EXISTS related_changes_recursive CASCADE;
 
+CREATE MATERIALIZED VIEW IF NOT EXISTS
+  config_item_analysis_change_count_7d AS
+WITH type_counts AS (
+    SELECT 
+        ca.config_id, 
+        ca.analysis_type, 
+        COUNT(*) AS type_count
+    FROM 
+        config_analysis ca
+    WHERE 
+        ca.status = 'open' AND
+        ca.last_observed >= NOW() - INTERVAL '7 days'
```

Comment:
I think we can remove the last_observed and then the index would be on config_id with a clause of status = 'open'

Reply:
So always aggregate all the analysis that are open ?